test/unit/http_utils_test.cpp

```diff
@@ -31,7 +31,7 @@
 #endif
 
 #include <cstdio>
-
+#include <filesystem>
```

Comment:
This requires C++17, not sure what @etr thinks about requiring C++17 in libhttpserver.  Current requirement is C++14 for the whole project.  Not sure if there should be different requirements for lib and tests?

Reply:
Oh, I missed that.
I just searched for the easiest portable way to check a file existence.

On linux this check would actually not be necessary, as mkstemp always creates a file and would fail, if it could not.
But on windows the creation of the file name and creation of the file are two separate steps in the function, so in my opinion this check should stay.

I will search for a easy C++14 way of doing that.

Reply:
thanks.

In general, I am not against a move to strict C++17, but this would be a bit small to justify that. Filesystem is doubly unfortunate because it would force C++ to version >8 and Clang >7.

In practice, you can check for file existence the same way it is done here: https://github.com/etr/libhttpserver/blob/master/src/file_response.cpp